index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <script charset="utf-8" src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.7.1/p5.min.js"></script>
+    <script charset="utf-8" src="javascript/main.js"></script>
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <body>
+    <div id="sketch-holder" style="text-align: center;"></div>
```

Comment:
styles should be in their own style css file

Reply:
I'm not sure why, but when I used it's own css with that style it wasn't working. But what I had it in-line it was, not sure why.

Reply:
scratch that, I forgot to link the css file.